tasties_app/tests/test_views.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+
+class TestViews:
+    @pytest.mark.django_db
+    def test_base_loaded(self, client):
+        response = client.get('/base/')
+        assert response.status_code == 200
+        assert b"tasties" in response.content
```

Comment:
A stray `b` right after the assert, is it a typo or intentional?

Reply:
Yes, it's intentional.
This `b` converts the string into byte-string. The reason I'm doing it is that `response.content` expects to get a byte-like object. If I'm sending a regular string it raises a TypeError:
![Screenshot 2023-01-01 132442](https://user-images.githubusercontent.com/85102059/210169177-422164ff-cedb-418e-acdd-bcf751ca33ce.png)